File: Nível 1/2868 - Errrou!/main.lua
quantia = tonumber(io.read())

for i = 1, quantia do 
    dados = io.read()
    x1, sinal, x2, igual, resultado = dados:match("(%S+)%s+(%S+)%s+(%S+)%s+(%S+)%s+(%S+)")
    x1, x2, resultado = tonumber(x1), tonumber(x2), tonumber(resultado)
    
    if sinal == "+" then 
        calc = x1 + x2 
    elseif sinal == "-" then
        calc = x1 - x2 
    elseif sinal == "x" then 
        calc = x1 * x2
    end

    if resultado < calc then
        n1 = calc 
        n2 = resultado
    elseif resultado >= calc then
        n1 = resultado
        n2 = calc 
    end

    calc2 = n1 - n2 

    errou = string.rep("r", calc2)
    print(string.format("E%sou!", errou))
end

File: Nível 1/2868 - Errrou!/main.py
quantia = int(input("A: "))

for i in range(1, quantia + 1):
    dados = list(map(str, input("B: ").split(" ")))

    if dados[1] == "+":
        calc = int(dados[0]) + int(dados[2])
    elif dados[1] == "-":
        calc = int(dados[0]) - int(dados[2])
    elif dados[1] == "x":
        calc = int(dados[0]) * int(dados[2])
    
    if int(dados[4]) < calc:
        x1 = calc
        x2 = int(dados[4])
    elif int(dados[4]) >= calc:
        x1 = int(dados[4])
        x2 = calc
    
    calc2 = x1 - x2 

    errou = "r" * calc2

    print(f"E{errou}ou!")